<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaTable</class>
 <widget class="QDialog" name="GammaTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GammaTable</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>7</y>
     <width>470</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;p align=&quot;center&quot;&gt;&lt;b&gt;&lt;font size=&quot;+1&quot;&gt;Dose Difference and DTA Table&lt;/font&gt;&lt;/b&gt;&lt;/p&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonHide">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>610</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Hide Passed</string>
   </property>
  </widget>
  <widget class="QLabel" name="textLabelRefIndex">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>640</y>
     <width>20</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>at</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxFiles">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>673</y>
     <width>220</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxCriteria">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>108</y>
     <width>491</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Criteria and Pass</string>
   </property>
   <widget class="QLabel" name="textLabelDD">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>24</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>DD (%)</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="textLabelDtaPass">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>60</y>
      <width>55</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Pass (%)</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="textLabelDdPass">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>55</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Pass (%)</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxDD">
    <property name="geometry">
     <rect>
      <x>69</x>
      <y>26</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="textLabelDTA">
    <property name="geometry">
     <rect>
      <x>159</x>
      <y>26</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>DTA (mm)</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxDTA">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>26</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDdPass">
    <property name="geometry">
     <rect>
      <x>78</x>
      <y>60</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDtaPass">
    <property name="geometry">
     <rect>
      <x>239</x>
      <y>60</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="textLabelPass">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>26</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>OR Pass (%)</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPass">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>26</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="textLabelGPass">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>60</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>G Pass (%)</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditGPass">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="textLabelRefDose">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>640</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Ref Dose (Gy)</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabelFile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>672</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Current File:</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxRefDose">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>640</y>
     <width>90</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Dref</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dmax</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pointwise</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEditRefDose">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>639</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPrint">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>649</y>
     <width>70</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>&amp;Print</string>
   </property>
   <property name="shortcut">
    <string>Alt+P</string>
   </property>
  </widget>
  <widget class="QLabel" name="dataLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>470</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;p align=&quot;center&quot;&gt;Machine&lt;/p&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabelBackground">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>609</y>
     <width>281</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Dose Background Cutoff (% of Ref Dose):</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>649</y>
     <width>70</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Clo&amp;se</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxCutoff">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>610</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableDD">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>481</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
